### PR TITLE
[EDGENT-327] Use local Javadoc URLs

### PR DESCRIPTION
**Pull request by queeniema**
- Removed all references to the DejaVu font in `stylesheet.css` during the site build process
- Added a new `favicon.ico` file to the root of the site

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 set -e
 cd site
 jekyll build -d ../content_tmp
+sed -i "" "/^@import/d; s/'DejaVu Sans',//g; s/'DejaVu Sans Mono',//g; s/'DejaVu Serif',//g" ../content_tmp/javadoc/*/stylesheet.css
```
Comment (ddebrunner) [inline build.sh]: Good to have a comment here as to why this (sed) is being done.

File: site/_config.yml [hunk 2]
```diff
@@ -126,9 +126,9 @@ host:    127.0.0.1
 sourcerepourl: https://github.com/apache/incubator-edgent
 downloadsurl: /docs/downloads
 projurl: /
-#docsurl: /javadoc/latest
+docsurl: /javadoc/latest
 # above yields things like the following when running on a "jekyll serve" test server
```
Comment (dlaboss) [inline site/_config.yml]: With your fixes these comments are no longer true so shouldn't they and the orig/wrong commented out docsurl should just be removed?

Comment (queeniema) [reply site/_config.yml]: Yes, they should be. I overlooked that. Thanks!

File: build.sh [hunk 3]
```diff
@@ -1,6 +1,8 @@
 set -e
 cd site
 jekyll build -d ../content_tmp
+# Remove dangling references to the DejaVu fonts in the Javadoc stylesheets
+sed -i "" "/^@import/d; s/'DejaVu Sans',//g; s/'DejaVu Sans Mono',//g; s/'DejaVu Serif',//g" ../content_tmp/javadoc/*/stylesheet.css
```
Comment (dlaboss) [inline build.sh]: Removing all imports vs only those associated with dejavu.css seems a little heavy handed?
Maybe also only have a single pattern to match any current, or future, DejaVu font spec?  Sed seems to lack a non-greedy specifier but the following seems to work s/'DejaVu[^']*',*//g  -- match everything starting with 'DejaVu and all chars up the first single quote, and that single quote, and an optional comma (i.e., if it came at the end of the font spec)

File: site/_config.yml (hunk 2 above)
Comment (dlaboss) [reply site/_config.yml]: woops, sorry, just noticed your jira comment that the script is run after the site is built hence the msgs will still show up when running "jekyll serve" https://issues.apache.org/jira/browse/EDGENT-327?focusedCommentId=15812317&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15812317
If that's the case maybe add some commentary to README.md (where it describes running "jekyll serve") so folks know its safe to ignore those particular error msgs?

**Comment by dlaboss** [comment]
+1 thanks for straightening this out!

**Pull request merged**